web/src/features/fba/components/infoPanel/AdvisoryText.tsx

```diff
@@ -127,6 +127,10 @@ const filterHFIStatsByArea = (stats: FireZoneFuelStats[]) => {
     stat => stat.fuel_area > FUEL_AREA_THRESHOLD || FUEL_TYPES_ALWAYS_INCLUDED.includes(stat.fuel_type.fuel_type_code)
   )
 }
+const SerifTypography = styled(Typography)({
+  fontSize: '1.2rem',
+  fontFamily: '"Courier New", "Courier", "Monospace" !important'
```

Comment:
Is `!important` required? What takes precedence over it?

Reply:
The fontFamily specified in index.css uses `!important` so I also need it here to override that one. We could look at setting the fontFamily in the theme instead in order to remove the need for `!important`.

Reply:
Nice thank you!